src/main/java/io/reactivex/Observable.java

```diff
@@ -450,35 +450,35 @@ public Throwable call() {
 
     public static <T> Observable<T> fromPublisher(final Publisher<? extends T> publisher) {
         Objects.requireNonNull(publisher, "publisher is null");
-        return create(new ObservableConsumable<T>() {
+        return new Observable<T>() {
```

Comment:
There should be another run to change such anonymous classes to proper internal classes.